src/test/scala/example/Codecs.scala

```diff
@@ -1,6 +1,5 @@
 package example
 
-
```

Comment:
Btw while we're at formatting, pls add .scalafmt.conf:

```
version = 2.6.4

style = defaultWithAlign

docstrings                 = JavaDoc
indentOperator.preset      = spray
maxColumn                  = 120
rewrite.rules              = [RedundantParens, AvoidInfix]
unindentTopLevelOperators  = true
align.tokens               = [{code = "=>", owner = "Case"}]
align.openParenDefnSite    = false
align.openParenCallSite    = false
optIn.breakChainOnFirstMethodDot = false
optIn.configStyleArguments   = false
danglingParentheses.defnSite = false
danglingParentheses.callSite = false
rewrite.neverInfix.excludeFilters = [
  and
  min
  max
  until
  to
  by
  eq
  ne
  "should.*"
  "contain.*"
  "must.*"
  in
  ignore
  be
  taggedAs
  thrownBy
  synchronized
  have
  when
  size
  only
  noneOf
  oneElementOf
  noElementsOf
  atLeastOneElementOf
  atMostOneElementOf
  allElementsOf
  inOrderElementsOf
  theSameElementsAs
  message
]
rewriteTokens = {
  "⇒": "=>"
  "→": "->"
  "←": "<-"
}
```

Reply:
Plus `scalafmtOnCompile := true` for the project (`import org.scalafmt.sbt.ScalafmtPlugin.autoImport.scalafmtOnCompile`)

Reply:
And `addSbtPlugin("org.scalameta" % "sbt-scalafmt" % <latest>)` to `project/plugins.sbt`